src/Compilers/Core/CommandLine/ConsoleUtil.cs

```diff
@@ -3,6 +3,8 @@
 using Microsoft.CodeAnalysis;
 using System;
 using System.IO;
+using System.Reflection;
+using System.Runtime;
```

Comment:
It looks like the only change is addition of usings. Consider reverting changes to this file, if the usings are not required.